devbox/src/syncer/SkipScanActor.scala

```diff
@@ -23,14 +23,18 @@ class SkipScanActor(sigActor: castor.Actor[SigActor.Msg],
   def initialState = Scanning(
     new PathSet(),
     new PathSet(),
-    mapping.map(_ => Skipper.fromString(ignoreStrategy)),
+    mapping.map { case (base, rel) => Skipper.fromString(ignoreStrategy, base) },
     0
   )
 
+  abstract class SkipperState(run0: SkipScanActor.Msg => State) extends State(run0) {
+    def skippers: Seq[Skipper]
```

Comment:
As mentioned elsewhere, we really shouldn't be peeking at the state of the actor from outside, and should instead instantiate the Skippers ahead of time, duplicate them for sending over the network, and then pass them to `SkipScanActor`'s constructor to initialize it